manifests/volume.pp

```diff
@@ -163,7 +163,7 @@
       # this volume exists
 
       # our fact lists bricks comma-separated, but we need an array
-      $vol_bricks = split( getvar( "::gluster_volume_${title}_bricks" ), ',')
+      $vol_bricks = getvar( "::gluster_volume_${title}_bricks" )
       if $bricks != $vol_bricks {
```

Comment:
I'd say it would be even better to sort the arrays already when generating the facts. I'll just try this now.